core/src/test/java/org/elasticsearch/search/aggregations/pipeline/BucketScriptIT.java

```diff
@@ -370,7 +423,13 @@ public void testInlineScriptInsertZeros() {
         }
     }
 
-    public void testIndexedScript() {
+    public void testStoredScript() {
+        assertAcked(client().admin().cluster().preparePutStoredScript()
+                .setId("my_script")
+                .setScriptLang(CustomScriptPlugin.NAME)
+                // Source is not interpreted but my_script is defined in CustomScriptPlugin
+                .setSource(new BytesArray("{ \"script\": \"_value0 + _value1 + _value2\" }")));
```

Comment:
This one is very misleading. It should be:

        assertAcked(client().admin().cluster().preparePutStoredScript()
                .setId("my_script")
                .setScriptLang(CustomScriptPlugin.NAME)
                // Script source is not interpreted but it references a pre-defined script from CustomScriptPlugin
                .setSource(new BytesArray("{ \"script\": \"my_script\" }")));